keps/sig-node/5018-dra-adminaccess/README.md

```diff
@@ -0,0 +1,687 @@
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+-->
+
+Will be considered for beta.
+
+### Scalability
+
+No.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No.
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No.
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+Will be considered for beta.
+
+###### What are other known failure modes?
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+Will be considered for beta.
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+Will be considered for beta.
+
+## Implementation History
+
+- Kubernetes 1.33: KEP accepted as "provisional".
+
+## Drawbacks
+
+None.
+
+## Alternatives
+
+The following options were also considered:
+
+- Validating Admission Policy (VAP): An example was added in 1.31. However this
+  approach cannot be used to control access for an in-tree type because
+  Kubernetes has no mechanism to apply a system VAP to all new clusters
+  automatically and therefore it is not sufficient for conformance.
+- Builtin admission controller: This is doable, but more work than the approach
+  described in this KEP.
```

Comment:
It is not more work, just different work.  David's comment has better reasoning to why admission is not the preferred approach.